app/src/main/java/org/p2p/wallet/home/ui/crypto/MyCryptoPresenter.kt

```diff
@@ -107,6 +112,17 @@ class MyCryptoPresenter(
         }
     }
 
+    private fun observePnlData() {
+        pnlDataSubscription?.cancel()
+        pnlDataSubscription = launch {
+            pnlInteractor.state
+                .onEach {
+                    handleTokenState(tokenServiceCoordinator.observeLastState().value)
```

Comment:
combine?

Reply:
they are synchronized only at the first launch and when forcibly refreshed; thereafter new values are produced at different times